### DISCUSSION TITLE
Request Transfer of Cake.CodeQL.Cli into cake-contrib Organization

### DISCUSSION THREAD

**Discussion by louisfischer** [cake-build/cake#3995] [Extension requests]
@cake-build-bot  I would like to request to transfer my new Addin for Cake into the Cake Contribution Organization.
Would someone be able to assist with this?

https://github.com/louisfischer/Cake.CodeQL.Cli

**Comment by nils-a** [comment]
Hey @louisfischer I can help.

If you transfer to my user, @nils-a, I will transfer it to the Cake Contrib organization

**Comment by louisfischer** [comment]
@nils-a Thanka. The repository has been transferred to you

**Reply by nils-a** [reply]
The Repo has been moved to [cake-contrib/Cake.CodeQL.Cli](https://github.com/cake-contrib/Cake.CodeQL.Cli/).
Let me know if you need anything else.

**Reply by louisfischer** [reply]
@nils-a thank you
